### ISSUE TITLE
"Missing credentials in config" "CredentialsError"

### ISSUE THREAD

**Issue by JacobJT** [acinader/parse-aws-firehose-logger-adapter#18]
I'm using both the latest version of parse-server and parse-aws-firehose-logger-adapter, and the setup worked fine from the ReadMe when I ran it on my local machine, but a collaborator tried to run the server locally and got the above error messages when using the adapter. Could you perhaps update the readme to show where to specify credentials?

**Comment by JacobJT** [comment]
I poked around in FirehoseLoggerAdapter.js and don't see anywhere that it's pulling credentials from, either. When we added the S3 adapter we had to specify some role credentials. Looking for how to do that here and am not seeing it.

**Comment by acinader** [comment]
Hi JacobJT, I added a note on AWS credentials to the documentation.

FWIW, I wrote the section of the S3 adapter readme :).

the key is to put your credentials either in ~/.aws/credentials or else create the appropriate environment vars on NON ec2 machines.  And then on ec2 use a role!  

The aws-sdk that really any node lib you find will depend on will always pick them up properly and using this best practice will increase your security and prevent you from inadvertently leaking credentials.

hope that helps, if not, feel free to post a question here.
